docs/pages/reference/authentication.mdx

```diff
@@ -121,6 +121,49 @@ $ tctl create -f cap.yaml
 </TabItem>
 </Tabs>
 
+### Local User Locks
```

Comment:
Per the style guide, we are using sentence case for headings. This should be "Local user locks"